Sources/Core/ReplaySubscription.swift

```diff
@@ -23,8 +24,14 @@ final class ReplaySubscription<Output, Failure>: Subscription where Failure: Err
         subscriber = nil
     }
 
-    func send(_ value: Output) {
-        process(buffer.append(value))
+    func append(_ value: Output) {
+        pendingValues += buffer.append(value)
+    }
+
+    func send() {
+        let values = pendingValues
+        pendingValues = []
+        process(values)
```

Comment:
```suggestion
        process(pendingValues)
        pendingValues = []
```

Reply:
Would of course be logical but does not work, see 2 in this [comment](https://github.com/SRGSSR/pillarbox-apple/issues/785#issuecomment-1987115101).